<ui version="4.0" >
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>543</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>680</width>
    <height>543</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>680</width>
    <height>543</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget" >
   <property name="geometry" >
    <rect>
     <x>4</x>
     <y>7</y>
     <width>671</width>
     <height>531</height>
    </rect>
   </property>
   <property name="currentIndex" >
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab" >
    <attribute name="title" >
     <string>Vendas</string>
    </attribute>
    <widget class="QLineEdit" name="edt_vlr_unit" >
     <property name="geometry" >
      <rect>
       <x>75</x>
       <y>193</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6" >
     <property name="geometry" >
      <rect>
       <x>80</x>
       <y>134</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>Descrição do Produto</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7" >
     <property name="geometry" >
      <rect>
       <x>6</x>
       <y>176</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>Qtd</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_vlr_total" >
     <property name="geometry" >
      <rect>
       <x>195</x>
       <y>193</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="btn_incluir" >
     <property name="geometry" >
      <rect>
       <x>315</x>
       <y>191</y>
       <width>33</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="icon" >
      <iconset>../../../../opt/kde/share/icons/crystalsvg/48x48/actions/down.png</iconset>
     </property>
     <property name="shortcut" >
      <string>Ins</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5" >
     <property name="geometry" >
      <rect>
       <x>6</x>
       <y>135</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>ID:</string>
     </property>
    </widget>
    <widget class="QToolButton" name="btn_excluir" >
     <property name="geometry" >
      <rect>
       <x>353</x>
       <y>191</y>
       <width>33</width>
       <height>28</height>
      </rect>
     </property>
     <property name="statusTip" >
      <string/>
     </property>
     <property name="text" >
      <string>...</string>
     </property>
     <property name="icon" >
      <iconset>../../../../opt/kde/share/icons/crystalsvg/48x48/actions/up.png</iconset>
     </property>
     <property name="shortcut" >
      <string>Del</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_qtd" >
     <property name="geometry" >
      <rect>
       <x>5</x>
       <y>193</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="focusPolicy" >
      <enum>Qt::StrongFocus</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_idpro" >
     <property name="geometry" >
      <rect>
       <x>5</x>
       <y>150</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="whatsThis" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>&lt;/body>&lt;/html></string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_nomepro" >
     <property name="geometry" >
      <rect>
       <x>80</x>
       <y>150</y>
       <width>251</width>
       <height>25</height>
      </rect>
     </property>
     <property name="inputMask" >
      <string/>
     </property>
     <property name="text" >
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_8" >
     <property name="geometry" >
      <rect>
       <x>76</x>
       <y>177</y>
       <width>68</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>Vlr_Unit</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9" >
     <property name="geometry" >
      <rect>
       <x>194</x>
       <y>176</y>
       <width>68</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>Vlr_Total</string>
     </property>
    </widget>
    <widget class="QToolButton" name="btn_findPro" >
     <property name="geometry" >
      <rect>
       <x>335</x>
       <y>150</y>
       <width>33</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text" >
      <string>...</string>
     </property>
     <property name="icon" >
      <iconset>../../../../opt/kde/share/icons/crystalsvg/48x48/apps/kdvi.png</iconset>
     </property>
    </widget>
    <widget class="QLabel" name="label_10" >
     <property name="geometry" >
      <rect>
       <x>547</x>
       <y>177</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>Vlr_T_Cmp</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3" >
     <property name="geometry" >
      <rect>
       <x>11</x>
       <y>80</y>
       <width>446</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit" >
     <property name="geometry" >
      <rect>
       <x>86</x>
       <y>30</y>
       <width>371</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>10</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>ID:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox" >
     <property name="geometry" >
      <rect>
       <x>501</x>
       <y>30</y>
       <width>86</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimumContentsLength" >
      <number>0</number>
     </property>
     <item>
      <property name="text" >
       <string>A Vista</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>A Prazo</string>
      </property>
     </item>
    </widget>
    <widget class="QToolButton" name="btn_findCli" >
     <property name="geometry" >
      <rect>
       <x>460</x>
       <y>30</y>
       <width>34</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text" >
      <string>...</string>
     </property>
     <property name="icon" >
      <iconset>../../../../opt/kde/share/icons/crystalsvg/48x48/apps/kdvi.png</iconset>
     </property>
    </widget>
    <widget class="QLabel" name="label_3" >
     <property name="geometry" >
      <rect>
       <x>501</x>
       <y>11</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>Venda Tipo</string>
     </property>
    </widget>
    <widget class="QLabel" name="label" >
     <property name="geometry" >
      <rect>
       <x>86</x>
       <y>10</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>Nome do Cliente:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4" >
     <property name="geometry" >
      <rect>
       <x>11</x>
       <y>61</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>Endereço:</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton" >
     <property name="geometry" >
      <rect>
       <x>598</x>
       <y>448</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="layoutDirection" >
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text" >
      <string>Avancar</string>
     </property>
     <property name="icon" >
      <iconset>../../../../opt/kde/share/icons/crystalsvg/48x48/actions/forward.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QTableWidget" name="grd_vendas" >
     <property name="geometry" >
      <rect>
       <x>0</x>
       <y>223</y>
       <width>661</width>
       <height>221</height>
      </rect>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="gridStyle" >
      <enum>Qt::SolidLine</enum>
     </property>
     <column>
      <property name="text" >
       <string>idpro</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>NomeProduto</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>qtd</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>vlr_unit</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>vlr_total</string>
      </property>
     </column>
    </widget>
    <widget class="QLineEdit" name="lineEdit_9" >
     <property name="geometry" >
      <rect>
       <x>541</x>
       <y>192</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2" >
    <attribute name="title" >
     <string>Crediário/ Parcelamento</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_3" >
    <attribute name="title" >
     <string>Concluir Vendas</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>btn_findCli</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>edt_idpro</tabstop>
  <tabstop>edt_nomepro</tabstop>
  <tabstop>btn_findPro</tabstop>
  <tabstop>edt_qtd</tabstop>
  <tabstop>edt_vlr_unit</tabstop>
  <tabstop>edt_vlr_total</tabstop>
  <tabstop>btn_incluir</tabstop>
  <tabstop>btn_excluir</tabstop>
  <tabstop>lineEdit_9</tabstop>
  <tabstop>grd_vendas</tabstop>
  <tabstop>toolButton</tabstop>
 </tabstops>
 <includes/>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX" >
   <number>10</number>
  </property>
  <property name="gridDeltaY" >
   <number>10</number>
  </property>
  <property name="gridSnapX" >
   <bool>true</bool>
  </property>
  <property name="gridSnapY" >
   <bool>true</bool>
  </property>
  <property name="gridVisible" >
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
